### PR TITLE
Documented the cache_control() decorator

### PR DESCRIPTION
**Pull request by marfire**
Also added the corresponding references to the cache framework
documentation.

### CODE REVIEW

File: docs/topics/cache.txt [hunk 1]
```diff
@@ -496,9 +496,10 @@ If a view sets its own cache expiry time (i.e. it has a ``max-age`` section in
 its ``Cache-Control`` header) then the page will be cached until the expiry
 time, rather than :setting:`CACHE_MIDDLEWARE_SECONDS`. Using the decorators in
 ``django.views.decorators.cache`` you can easily set a view's expiry time
-(using the ``cache_control`` decorator) or disable caching for a view (using
-the ``never_cache`` decorator). See the `using other headers`__ section for
-more on these decorators.
+(using the :func:`~django.views.decorators.cache.cache_control()` decorator) or
```
Comment (timgraham) [inline docs/topics/cache.txt]: The parentheses aren't needed -- Sphinx adds them automatically in the rendered docs.

File: docs/topics/cache.txt [hunk 2]
```diff
@@ -496,9 +496,10 @@ If a view sets its own cache expiry time (i.e. it has a ``max-age`` section in
...
+disable caching for a view (using the :func:`~django.views.decorators.cache.never_cache()`
```
Comment (timgraham) [inline docs/topics/cache.txt]: Please wrap lines at 79 characters.

File: docs/topics/http/decorators.txt [hunk 3]
```diff
@@ -111,3 +111,9 @@ client-side caching.
     This decorator adds a ``Cache-Control: max-age=0, no-cache, no-store,
     must-revalidate`` header to a response to indicate that a page should never
     be cached.
+
+.. function:: cache_control(**kwargs)
```
Comment (timgraham) [inline docs/topics/http/decorators.txt]: Consider putting this before never_cache for alphabetization purposes.

**Comment by marfire** [comment]
I'm not seeing that build error locally. Seems to be about something else?

**Comment by timgraham** [comment]
Merged in b785927b442d1e4b59f2afa40729a4ca1a7ed3ab, thanks! 

p.s. You might want to configure your editor to remove trailing whitespace on save, I noticed and removed some.

**Comment by timgraham** [comment]
Looks like a transient network issue where sphinx couldn't download from readthedocs.

**Comment by marfire** [comment]
Noted on the whitespace. I didn't bother using my "real" editor for these text files, but I will next time.

**Pull request closed without merging**